plugin/pxr/maya/lib/pxrUsdMayaGL/testenv/testProxyShapeDuplicatePerformance.py

```diff
@@ -15,6 +15,8 @@
 # limitations under the License.
 #
 
+from future.utils import iteritems
```

Comment:
Is the `iteritems(cls._profileScopeMetrics)` change from testProxyShapeDrawPerformance.py above missing in `tearDownClass()` below?

These tests don't yet run with CMake/CTest, so I don't expect that they'd show up in your test results.

Reply:
Hmm, I missed one???  See e594505ae0d7ba2258f47edfb48bad0b558c2424

Correct, it looks like we are ignoring the tests in pxrUsdMayaGL and that is why I didn't see this error.

Anyways, all tests in usdMaya pass successful except one that I mentioned earlier.

![image](https://user-images.githubusercontent.com/48299423/83581989-42610080-a50e-11ea-98ee-184842a5ae09.png)